src/sidebar/helpers/view-filter.js

```diff
@@ -120,87 +94,90 @@ class BinaryOpFilter {
 }
 
 /**
- * Functions for extracting field values from annotations and testing whether
- * they match a query term.
+ * Create a matcher that tests whether a query term appears anywhere in a
+ * string field value.
  *
- * [facet_name]:
- *   autofalse: a function for a preliminary false match result
- *   value: a function to extract to facet value for the annotation.
- *   match: a function to check if the extracted value matches the facet value
+ * @param {(ann: Annotation) => string[]} fieldValues
+ * @return {Matcher}
+ */
+function stringFieldMatcher(fieldValues) {
+  return {
+    fieldValues,
+    matches: (value, term) => value.includes(term),
+    normalize: normalizeStr,
+  };
+}
+
+/**
+ * Map of field name (from a parsed query) to matcher for that field.
  *
- * @type {Record<string,Checker>}
+ * @type {Record<string, Matcher|Matcher<number>>}
```

Comment:
Can you help me understand the need for the separately-delineated `Matcher<number>` here? Is it because of the `since` property?

Reply:
Yes, exactly.